src/breadcrumb-group/implementation.tsx

```diff
@@ -68,6 +75,15 @@ const EllipsisDropdown = ({
         onItemClick={onDropdownItemClick}
         onItemFollow={onDropdownItemFollow}
         customTriggerBuilder={getDropdownTrigger}
+        analyticsMetadataTransformer={metadata => {
+          if (metadata.detail?.id) {
+            delete metadata.detail.id;
```

Comment:
The `id` field makes sense in button dropdown, because it is provided by consumers. In this case, we use the button dropdown internally to display breadcrumbs, and auto-generate those ids. So they do not bring value as analytics-related information.